### PR TITLE
Dev aug2016  For August 2016 release

### PR DESCRIPTION
**Pull request by ronaldddilley**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,41 +1,42 @@
...
 ### Installation of Specific Algorithms
-Please see the installation instructions within the algorithm sub-directory.
+Please see the installation instructions for the algorithm.
+- [cfmask](cfmask)
 
 ### Installation of All Algorithms
 
 ### Dependencies
-* ESPA raw binary libraries, tools, and it's dependencies, found here [espa-product-formatter](https://github.com/USGS-EROS/espa-product-formatter)
+- ESPA raw binary libraries, tools, and it's dependencies, found here [espa-product-formatter](https://github.com/USGS-EROS/espa-product-formatter)
```
Comment (raydittmeier) [inline README.md]: nitpicking: "its" instead of "it's"

File: cfmask/README.md [hunk 2]
```diff
@@ -11,19 +11,16 @@ CFMASK (Function of Mask Algorithm).
...
-* A bug fix to the cloud labeling code.
-* Lots of general code cleanup.
+- Fix to the confidence band to more accurately reflect confidence in scenes with greater-than 90% cloud cover.  Previously this condition was not being set in the confidence band.
+- Modification to bring up-to-date with FMask 3.3.1.  Use a new "if" to determine whether an object is smaller than 3 pixels.  If it is smaller, use the minimum temperature.
 
 ## Installation
 
 ### Dependencies
-* ESPA raw binary libraries, tools, and it's dependencies, found here [espa-product-formatter](https://github.com/USGS-EROS/espa-product-formatter)
+- ESPA raw binary libraries, tools, and it's dependencies, found here [espa-product-formatter](https://github.com/USGS-EROS/espa-product-formatter)
```
Comment (raydittmeier) [inline cfmask/README.md]: "its" instead of "it's"

File: cfmask/src/object_cloud_shadow_match.c [hunk 3]
```diff
@@ -746,28 +746,35 @@ int object_cloud_shadow_match
...
 
                 /* number of inward pixels for correct temperature */
-                pct_obj = ((cloud_radius - num_pix) * (cloud_radius - num_pix))
-                          / (cloud_radius * cloud_radius);
-                if (pct_obj >= 1.0)
+                if (cloud_radius > num_pix)
                 {
-                    /* Use the minimum temperature instead */
-                    t_obj = temp_obj_min;
+                    pct_obj = ((cloud_radius - num_pix)
```
Comment (raydittmeier) [inline cfmask/src/object_cloud_shadow_match.c]: I really doubt the optimized compiler would miss this, but the cloud_radius - num_pix calculation is done twice, so you could do it once and put the value in a variable, and use that variable here instead.

File: README.md (hunk 1 above)
Comment (ronaldddilley) [inline README.md]: fixed

File: cfmask/README.md (hunk 2 above)
Comment (ronaldddilley) [inline cfmask/README.md]: fixed

File: cfmask/src/object_cloud_shadow_match.c (hunk 3 above)
Comment (ronaldddilley) [inline cfmask/src/object_cloud_shadow_match.c]: I remember thinking that, and then decided to let the compiler worry about it.

**Comment by gschmidt-usgs** [comment]
Peer review complete. No comments.

**Pull request merged**